packages/validator/transpile.test-d.ts

```diff
@@ -1,5 +1,5 @@
 import { expectType } from "tsd";
-import type { transpileLocale, transpileSchema } from "./transpile.t.ds";
```

Comment:
These are obviously not types. The problem is that current TSConfig defaults to `moduleResolution: "node10"` and that on allows oddities like this one.